src/main/java/emissary/config/ConfigUtil.java

```diff
@@ -568,26 +562,23 @@ public static InputStream getConfigData(final String f) throws IOException {
     public static Configurator getMasterClassNames() throws IOException, EmissaryException {
         final List<File> masterClassNames = new ArrayList<>();
         for (final String dir : getConfigDirs()) {
-            final File[] files = new File(dir).listFiles(new FilenameFilter() {
-                @Override
-                public boolean accept(final File dir, final String name) {
-                    return name.startsWith("emissary.admin.MasterClassNames") && name.endsWith(".cfg");
-                }
-            });
+            final File[] files = new File(dir).listFiles((dir1, name) -> name.startsWith("emissary.admin.MasterClassNames") && name.endsWith(".cfg"));
             // sort the files, to put emissary.admin.MasterClassNames.cfg before emssary.admin.MasterClassNames-blah.cfg
-            Arrays.sort(files);
-            masterClassNames.addAll(Arrays.asList(files));
+            if (files != null) {
+                Arrays.sort(files);
+                masterClassNames.addAll(Arrays.asList(files));
+            }
         }
         // check to make sure we have at least one
         // TODO make a test for this
```

Comment:
Yep. We still need a test for this!

Reply:
https://github.com/NationalSecurityAgency/burrito-grande/issues/1920